diff --git a/templates/amu-admin/src/layouts/AdminLayout.vue b/templates/amu-admin/src/layouts/AdminLayout.vue
--- a/templates/amu-admin/src/layouts/AdminLayout.vue
+++ b/templates/amu-admin/src/layouts/AdminLayout.vue
@@ -190,13 +190,11 @@
         </div>
 
-        <div class="admin-layout__scrollbar">
-          <AmuScrollbar>
-            <div class="admin-layout__view">
-              <RouterView v-slot="{ Component, route: currentRoute }">
-                <Transition name="fade-transform" mode="out-in">
-                  <KeepAlive :include="aliveCacheNames">
-                    <component :is="Component" :key="\\::\\" />
-                  </KeepAlive>
-                </Transition>
-              </RouterView>
-            </div>
-          </AmuScrollbar>
-        </div>
+        <AmuScrollbar class="admin-layout__scrollbar">
+          <div class="admin-layout__view">
+            <RouterView v-slot="{ Component, route: currentRoute }">
+              <Transition name="fade-transform" mode="out-in">
+                <KeepAlive :include="aliveCacheNames">
+                  <component :is="Component" :key="\\::\\" />
+                </KeepAlive>
+              </Transition>
+            </RouterView>
+          </div>
+        </AmuScrollbar>
       </section>
